### ISSUE TITLE
A group of network pictures, one does not show.

### ISSUE THREAD

**Issue by hkllzh** [facebook/fresco#2326]
The url of the image that cannot be displayed is

```
https://qn.yixionline.com/enjoy/app/dev/children/enjoyShow/image/2019-04-17_16-51-08_348
```

Fresco init config
``` kotlin
val pipelineConfig = OkHttpImagePipelineConfigFactory
                .newBuilder(this, HttpServer.okHttpClient)
```

**Comment by zmroczek** [comment]
Hi @hkllzh 

Could you provide a sample code how you are loading the image or did you try to reproduce it in the sample app?

**Comment by hkllzh** [comment]
Hi @zmroczek 

Here's a simple test App.
The first image can be loaded, but the second cannot.
Consistent with our performance in the official App.

app zip url：`https://image.yixionline.com/test/app/TestFrescoLoadPic.zip`

**Comment by hkllzh** [comment]
The reason I found this image to be undisplayable is the exif exception in the image.
```
Orientation: {"val":"90","type":3}
```
My current solution is to not rotate the image. However, the normal picture of this attribute is displayed in the wrong direction.
I hope this information can help you.
I want to display this type of image using Fresco's default configuration.

**Comment by defHLT** [comment]
Hey hkllzh, thanks for following through with this issue. You are right about Exif. The image is not displayed because valid values for orientation is 1 to 8, but it's set to 90. We probably should ignore this invalid value and show the image without trying to rotate it.

**Comment by defHLT** [comment]
Fixed in a0c09d12709f26e7079c874b6e6c607364678b1c

**Comment by hkllzh** [comment]
thank you
